src/main/java/dao/impl/JDBC/BasketDaoJDBC.java

```diff
@@ -0,0 +1,104 @@
+package dao.impl.JDBC;
+
+import dao.BasketDao;
+import model.Basket;
+import model.Product;
+import model.User;
+import org.apache.log4j.Logger;
+import utils.ConnectionJDBC;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class BasketDaoJDBC implements BasketDao {
+
+    private Logger logger = Logger.getLogger(OrderDaoJDBC.class);
+
+    @Override
+    public void addBasket(User user) {
+        try (Connection connection = ConnectionJDBC.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(
+                    "INSERT INTO baskets (user_id) VALUES (?)");
```

Comment:
better move queries from method to class

Reply:
Is it neccessary ? I think it will be not handy..

Reply:
maybe, but it look like more understandable when query is placed separately from the PreparedStatement